int_tests/vectara_int_tests/managers/test_corpora.py

```diff
@@ -0,0 +1,188 @@
+        self.assertEqual(response.key, "test-update-corpus")
+        self.assertEqual(response.name, "test-update-corpus")
+
+        response = self.client.corpora.update("test-update-corpus",
+                                              name="updated-name", description="updated-description")
+
+        time.sleep(30)
+
+        self.assertEqual(response.description, "updated-description")
+        self.assertEqual(response.name, "updated-name")
+
+    def test_get_metadata_of_corpora(self):
+        self.client.corpora.create(key="test-get-metadata",
+                                   description="test-description", name="Test")
+        time.sleep(30)
+        corpora = self.client.corpora.get("test-get-metadata")
+
+        self.assertEqual(corpora.key, "test-get-metadata")
+        self.assertEqual(corpora.name, "Test")
+        self.assertEqual(corpora.description, "test-description")
+
+    def test_corpus_reset(self):
+        self.client.corpora.create(key="test-reset-corpus")
+        time.sleep(30)
+        document = CoreDocument(
+            id="my-doc-id",
+            document_parts=[
+                CoreDocumentPart(
+                    text="I'm a nice document part.",
+                )
+            ],
+        )
+        self.client.documents.create("test-reset-corpus", request=document)
+        documents = self.client.documents.list("test-reset-corpus")
+        self.assertEqual(len(list(documents)), 1)
+
+        self.client.corpora.reset("test-reset-corpus")
+
+        documents = self.client.documents.list("test-reset-corpus")
+        self.assertEqual(len(list(documents)), 0)
+
+    def test_replace_filter_attributes(self):
+        self.client.corpora.create(key="test-reset-filters")
+        time.sleep(30)
+        filter_attributes = FilterAttribute(
+            name="Title",
+            level="document",
+            description="The title of the document.",
+            indexed=True,
+            type="text"
+        )
+        res = self.client.corpora.replace_filter_attributes("test-reset-filters", filter_attributes=[filter_attributes])
+
+        self.assertIsNotNone(res.job_id)
+
+    def test_search(self):
+        self.client.corpora.create(name="test-search", key="test-search")
+        time.sleep(30)
+        document = CoreDocument(
+            id="my-doc-id",
+            document_parts=[
+                CoreDocumentPart(
+                    text="Robot Utility Models are trained on a diverse set of environments and objects, and then can "
+                         "be deployed in novel environments with novel objects without any further data or training.",
+                )
+            ],
+        )
+        self.client.documents.create("test-search", request=document)
+
+        response = self.client.corpora.search(corpus_key="test-search", query="Robot Utility Models")
+        self.assertIsNone(response.summary)
+        self.assertGreater(len(response.search_results), 0)
```

Comment:
Shouldn't it give some result?

Reply:
summary and search results are the results that are used further along with other things.